cluster/cluster.go

```diff
@@ -331,3 +333,28 @@ func sizeToBytes(s string) (uint64, error) {
 	}
 	return 0, fmt.Errorf("'%s' format error, must be a positive integer with a unit of measurement like K, M or G", s)
 }
+
+// IsMysqlClusterKind for the given kind checks if CRD kind is for Cluster CRD.
+func IsMysqlClusterKind(kind string) bool {
```

Comment:
funciton Name need change? where it be called？

Reply:
This function is not used in pr this time, I will remove it.